### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by cutieprobe**
Add screenshots for command suggestion and some meme commands.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -27,7 +27,7 @@ Weme is a meme manager app for those who *prefer to use a desktop app for managi
 .  Copy the file to the folder you want to use as the home folder for your Weme.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
 +
-//image::Ui.png[width="790"]
+image::../Ui.png[width="790"]
```
Comment (moziliar) [inline docs/UserGuide.adoc]: Can't see anything.

Comment (gary-lgy) [inline docs/UserGuide.adoc]: Wrong path

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -103,6 +103,12 @@ Format: `tab TAB_NAME` +
 * *import*
 * *preferences*
 
+Example:
+
+* Switch to `templates` tab using `tab templates` command:
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]:
```suggestion
* `tab templates` +
 Switches to the `templates` tab.
```

Please ensure consistency with the rest of the user guide.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (gary-lgy) [reply docs/UserGuide.adoc]: Remove the leading `../`, please test before push

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -123,6 +129,16 @@ Examples:
...
+Example of adding a meme with description and tags: +
+`add p/pathToMeme d/Cat and child t/baby t/cuteCate`
+
+Before command:
+
+image::MemeAddCommand2.png[width="790"]
+After command:
+
+image::MemeAddCommand.png[width="790"]
+
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]: Put these after the box explaining relative and absolute paths.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -123,6 +129,16 @@ Examples:
 * `add p//Users/Me/Downloads/dgirl_oof.jpg t/Funny` +
 Adds a meme with the image from `/Users/Me/Downloads/dgirl_oof.jpg` and tag `Funny`
 
+Example of adding a meme with description and tags: +
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]:
```suggestion
Illustration of adding a meme at `pathToMeme` with description `Cat and child` and tags `baby` and `cuteCat`: +
```

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -163,6 +179,15 @@ Edits the description of the 1st meme to be `Funny SoC Meme` and tags to be `fun
 * `edit 2 d/Random Meme t/` +
 Edits the description of the 2nd meme to be `Random Meme` and clears all existing tags.
 
+Example of editing a meme's description: +
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]:
```suggestion
Illustration of editing the description of the meme at index 1 in the displayed list: +
```

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -182,6 +207,15 @@ Examples:
 * `find pikachu` +
 Returns memes whose tags contain `pikachu`
 
+Example of finding memes with a tag +
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]:
```suggestion
Illustration of finding memes with tag `cute` +
```

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -182,6 +207,15 @@ Examples:
...
+`find cute`
+
+After command:
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]: Missing before command screenshot

**Pull request merged**